packages/medusa/src/repositories/money-amount.ts

```diff
@@ -1,5 +1,55 @@
-import { EntityRepository, Repository } from "typeorm"
+import { Brackets, EntityRepository, In, IsNull, Not, Repository } from "typeorm"
 import { MoneyAmount } from "../models/money-amount"
 
+type Price = Partial<Pick<MoneyAmount, 'currency_code' | 'region_id' | 'sale_amount' | 'currency_code'>> & { amount: number }
+
 @EntityRepository(MoneyAmount)
-export class MoneyAmountRepository extends Repository<MoneyAmount> { }
+export class MoneyAmountRepository extends Repository<MoneyAmount> {
+  public async findVariantPricesNotIn(
+    variantId: string,
+    prices: Price[]
+  ) {
+    const pricesNotInPricesPayload = await this.createQueryBuilder(
+    )
+      .where({
```

Comment:
nit: Don't know if your linter is acting up here?? :D

Reply:
it was :sob: